build-system/compile/bundles.config.js

```diff
@@ -224,6 +224,10 @@ exports.extensionAliasBundles = {
     version: '1.0',
     aliasedVersion: '0.1',
   },
+  'amp-mustache': {
```

Comment:
i think this file is for extensions in src/bento/components, which bento-mustache isn't (for now)

Reply:
Ack. Removed.